cypress/integration/toynet/first.spec.ts

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
Lets break each one of these tests out into their own files. For example, we could have `HomePage.spec.ts` for the `describe` in line 27.